src/Components/CreditCardInput/CreditCardInput.tsx

```diff
@@ -50,30 +61,74 @@ export const CreditCardInput: React.FC<CreditCardInputProps> = ({
 }) => {
   return (
     <Box data-test="creditCardInput" {...rest}>
-      <Flex flex={1}>
-        <Join separator={<Spacer x={2} />}>
-          <CardElementContainer flex={3} error={!!error}>
-            <CardNumberElement
-              onChange={onChange}
-              options={{ showIcon: true, style: { base: stripeBaseStyle } }}
-            />
-          </CardElementContainer>
+      <Media greaterThanOrEqual="md">
+        <GridColumns>
+          <Column span={12}>
+            <Flex flex={1}>
+              <Join separator={<Spacer x={2} />}>
+                <CardElementContainer flex={3} error={!!error}>
+                  <CardNumberElement
+                    onChange={onChange}
+                    options={{
+                      showIcon: true,
+                      style: { base: stripeBaseStyle },
+                    }}
+                  />
+                </CardElementContainer>
+
+                <CardElementContainer flex={1} error={!!error}>
+                  <CardExpiryElement
+                    onChange={onChange}
+                    options={{ style: { base: stripeBaseStyle } }}
+                  />
+                </CardElementContainer>
+
+                <CardElementContainer flex={1} error={!!error}>
+                  <CardCvcElement
+                    onChange={onChange}
+                    options={{ style: { base: stripeBaseStyle } }}
+                  />
+                </CardElementContainer>
+              </Join>
+            </Flex>
+          </Column>
+        </GridColumns>
+      </Media>
```

Comment:
**Suggestion:** We could use `Column` to have different values instead of using the `Media` component like this:

```tsx
...
	<Column span={[12, 6]}>
...
```

Would you mind if I update it to remove the `Media` component?

Reply:
Sure - will do. thanks for reviewing this